cumulusci/robotframework/CumulusCI.py

```diff
@@ -245,14 +245,29 @@ def debug(self):
         set_pdb_trace()
 
     def start_perf_timer(self):
-        """Start an elapsed time stopwatch for performance tests"""
+        """Start an elapsed time stopwatch for performance tests.
+
+        Example:
+
+            Start Perf Timer
+            Do Something
+            End Perf Timer
+        """
         BuiltIn().set_test_variable("${__start_time}", datetime.now())
 
     def end_perf_timer(self):
```

Comment:
I mentioned this in another comment, but you might want to consider renaming this from `end_` to `stop_`, if for no other reason that it will appear right after the documentation for the `start_` keyword.

Reply:
Renamed.